chainercv/evaluations/eval_semantic_segmentation.py

```diff
@@ -77,30 +86,31 @@ class labels as values, which is obtained from inference.
         the number of images in the input.
```

Comment:
How about list of arrays or `N, H, W`.
`N, H, W` is probably the most common usage.